cmd/compute-domain-kubelet-plugin/driver.go

```diff
@@ -147,7 +146,7 @@ func (d *driver) PrepareResourceClaims(ctx context.Context, claims []*resourceap
 }
 
 func (d *driver) UnprepareResourceClaims(ctx context.Context, claimRefs []kubeletplugin.NamespacedObject) (map[types.UID]error, error) {
-	klog.Infof("UnprepareResourceClaims called with %d claim(s)", len(claimRefs))
+	klog.V(6).Infof("UnprepareResourceClaims called with %d claim(s)", len(claimRefs))
```

Comment:
Note that we typically see pairs of these log messages:
```
I0513 12:35:44.198425       1 driver.go:149] UnprepareResourceClaims called with 1 claim(s)
I0513 12:35:44.204757       1 driver.go:210] unprepared devices for claim 'nvidia-dra-driver-gpu/imex-channel-injection-rmr8b-zh5nx-compute-domain-daemon-xhlq2:c515582f-eb9d-40c4-b6f3-3729110b4755'
```

Instead of removing the first msg, I for now just wanted to mark it as more 'debuggy' (less serious/important).